examples/misc/multiprocess.py

```diff
@@ -8,9 +8,8 @@
 import numpy as np
 
 import matplotlib
-matplotlib.use('GtkAgg')
+matplotlib.use('TkAgg')
```

Comment:
Since you're using canvas API now, this might be portable enough to not require setting a specific backend at all.